src/components/Hero.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+
+function Hero() {
+  return (
+    <div className="Hero">
```

Comment:
is `.Hero` targeted in css? Also, generally HTML class names are not capitalised.